### PR TITLE
Fix website links and use common library for PHPMD and Pdepend

### PR DESCRIPTION
**Pull request by kylekatarnls**
This need first #623 to be merged and this to be rebased on master.

### CODE REVIEW

**Comment by kylekatarnls** [comment]
I have to check page content handling how to make absolute links working: https://github.com/phpmd/phpmd/pull/631

File: src/site/rst/about.rst [hunk 1]
```diff
@@ -3,13 +3,10 @@ PHPMD - PHP Mess Detector
 =========================
 
 This is the project site of *PHPMD*. It is a spin-off project of
-`PHP Depend`__ and aims to be a PHP equivalent of the well known
-Java tool `PMD`__. PHPMD can be seen as an user friendly and easy
+`PHP Depend <http://pmd.sourceforge.net>`_ and aims to be a PHP equivalent of the well known
```
Comment (tvbeek) [inline src/site/rst/about.rst]: I think you switched the urls :)

Comment (kylekatarnls) [reply src/site/rst/about.rst]: Fixed.

**Comment by kylekatarnls** [comment]
@tvbeek @ravage84 Is there something to change here?

**Comment by tvbeek** [comment]
> 
> 
> @tvbeek @ravage84 Is there something to change here?

No, just missed that I need to re review it 😃

**Comment by kylekatarnls** [comment]
Use #711 instead

**Pull request closed without merging**